server/events.go

```diff
@@ -1005,10 +1053,14 @@ func (s *Server) addSystemAccountExports(sacc *Account) {
 	if !s.EventsEnabled() {
 		return
 	}
-	accConnzSubj := fmt.Sprintf(accReqSubj, "*", "CONNZ")
+	accConnzSubj := fmt.Sprintf(accDirectReqSubj, "*", "CONNZ")
 	if err := sacc.AddServiceExportWithResponse(accConnzSubj, Streamed, nil); err != nil {
 		s.Errorf("Error adding system service export for %q: %v", accConnzSubj, err)
 	}
+	accStatzSubj := fmt.Sprintf(accDirectReqSubj, "*", "STATZ")
```

Comment:
This is weird.. from reading more in the PR it looks like the account is located in index 3 (4th token), which then can be "PING" in accPingReqSubj. So do we have situations where "PING.STATZ" we don't know if it is a PING or it prevents account named "PING"?

Reply:
Yes this happens, which is why there is a filter for ping.
This happens for server too, there it's not happening as every server subscribes to the exact name.
Not so for accounts, which is why this filter got added.
```
} else if acc == "PING" { // Filter ping subject, this happens for server too, but filter is less explicit
```

Reply:
This is the same for server and it's ping counter part.
There it's not observed as each server subscribes to it's ID, and is not using a wildcard.
STATZ call for an account filters when the name is PING